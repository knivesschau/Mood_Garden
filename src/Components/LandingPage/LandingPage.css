.LandingPage {
    text-align: center;
}

.Description_Intro, .Description_Functional, .Description_Final {
    padding: 1.5em;
    margin: 1em;
    min-height: 200px;
    border: 1px solid black;
}

p {
    margin-left: 3em;
    margin-right: 3em;
}