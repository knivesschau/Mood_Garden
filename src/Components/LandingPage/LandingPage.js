import React, { Component } from 'react';
import RoseColor from '../RoseColor/RoseColor';
import MainNav from '../MainNav/MainNav';
import './LandingPage.css';

export default class LandingPage extends Component {
    render() {
        return (
            <div className="LandingPage">
                
                <nav role="navigation" className="Landing_Nav">
                    <MainNav/>
                </nav>

                <header role="banner">
                    <h1>Mood Garden</h1>
                    
                    <h2 id="pitch-header">Cultivate Your Mind</h2>
                </header>

                <div className="Description_Intro">
                    
                    <RoseColor/>
                    
                    <h3 id="tagline1">Cultivate your mindfulness, one day at a time.</h3>

                    <p id="landing1">Mood Garden allows you to learn, grow, and reflect on your life by cultivating a rose garden that acts as your daily mood journal.</p>
                
                </div>

                <div className="Description_Functional">
                    
                    <RoseColor/>
                    
                    <h3 id="tagline2">Plant your Roses</h3>

                    <p id="landing2">Fill your garden with roses that act as daily reflections of your activities, thoughts, and actions. Mood Garden guides you on cultivating your garden with reflective and insightful prompts.</p>
                
                </div>

                <div className="Description_Final">
                    
                    <RoseColor/>
                    
                    <h3 id="tagline3">Walk in Your Garden</h3>

                    <p id="landing3">Take a walk through your mind's garden and reflect on your past journal entries anytime, anywhere. You may even come up with more insights!</p>

                </div>

            </div>
                        
        );
    }
}