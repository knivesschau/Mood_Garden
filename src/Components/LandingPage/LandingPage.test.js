import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import LandingPage from './LandingPage';

describe ('LandingPage Component', () => {
    it ('renders the Complete LandingPage Component and all nested Components', () => {
        const wrapper = shallow(<LandingPage/>);
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})