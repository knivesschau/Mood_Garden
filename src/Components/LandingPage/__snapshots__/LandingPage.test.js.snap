// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage Component renders the Complete LandingPage Component and all nested Components 1`] = `
<div
  className="LandingPage"
>
  <nav
    className="Landing_Nav"
    role="navigation"
  >
    <MainNav />
  </nav>
  <header
    role="banner"
  >
    <h1>
      Mood Garden
    </h1>
    <h2>
      Cultivate Your Mind
    </h2>
  </header>
  <div
    className="Description_Intro"
  >
    <h3>
      Cultivate your mindfulness, one day at a time.
    </h3>
    <p>
      Mood Garden allows you to learn, grow, and reflect on your life by cultivating a rose garden that acts as your daily mood journal.
    </p>
  </div>
  <div
    className="Description_Functional"
  >
    <h3>
      Plant your roses
    </h3>
    <p>
      Each day, you can plant roses into your mood journal and reflect on your daily activities, thoughts, and actions. Mood Garden guides you on cultivating your garden with reflective and insightful prompts.
    </p>
  </div>
  <div
    className="Description_Final"
  >
    <h3>
      Walk in your garden
    </h3>
    <p>
      Take a walk through your mind's garden and reflect on your past journal entries anytime, anywhere.
    </p>
  </div>
</div>
`;
